config/locales/server.en.yml

```diff
@@ -561,6 +565,11 @@ en:
           attributes:
             word:
               too_many: "Too many words for that action"
+        bookmark:
+          attributes:
+            reminder_at:
+              blank: "time must be provided for all reminders except 'Next time I'm at my Desktop'"
```

Comment:
It might be a good idea to replace `Next time I'm at my Desktop` with a placeholder like `%{reminder_name}` to ensure that the name doesn't get out of sync from the actual translation of `bookmarks.reminders.at_desktop`. There's already a minor difference right now. 😉